README.md

```diff
@@ -6,6 +6,29 @@ ExactTarget Fuel SDK / SalesforceMarektingCloudSDK for Ruby
 ## Overview ##
 The Fuel SDK for Ruby provides easy access to ExactTarget's Fuel API Family services, including a collection of REST APIs and a SOAP API. These APIs provide access to ExactTarget functionality via common collection types such as array/hash.
 
+
+## New Features in Version 1.2.0 ##
+- **OAuth2 authentication support** - [More Details](https://developer.salesforce.com/docs/atlas.en-us.mc-app-development.meta/mc-app-development/integration-considerations.htm)
+
+    To enable OAuth2 authentication, pass ```use_oAuth2_authentication => true``` in the params argument to the Client's class constructor.
+    
+    Example of instantiating the Client class:
+    
+    ```
+    myclient = MarketingCloudSDK::Client.new({
+        'client' => {
+            'id' => '<CLIENT_ID>',
+            'secret' => '<CLIENT_SECRET>',
+            'request_token_url' => '<AUTH TENANT SPECIFIC ENDPOINT>',
+            'soap_endpoint' => '<SOAP TENANT SPECIFIC ENDPOINT>',
+            'base_api_url' => '<REST TENANT SPECIFIC ENDPOINT>',
+            'use_oAuth2_authentication' => true,
+            'account_id' => <TARGET_ACCOUNT_ID>,
+            'scope' => '<PERMISSION_LIST>'
+            }
+        })
+    ```
+
```

Comment:
we also need to tell them that they can send the endpoint information through parameters of client class as well.

Reply:
The Ruby SDK supports only constructor parameters for the settings. The samples appear to use a config file, but that is a plain .rb file which declares a json object with all the settings. The file is than imported and that json object is mapped to the params constructor parameter.